crates/rpc/rpc-api/src/engine.rs

```diff
@@ -257,6 +257,7 @@ pub trait EngineEthApi {
         &self,
         number: BlockNumberOrTag,
         full: bool,
+        include_header: Option<bool>,
```

Comment:
lets be more specific
```suggestion
        include_extra_data_header: Option<bool>,
```

Reply:
Done!